pages/dkp/kommander/2.3/install/configuration/enterprise-catalog/index.md

```diff
@@ -55,7 +55,7 @@ When running in air-gapped environments, update the configuration by replacing `
 1.  Download the DKP catalog application Git repository archive:
 
     ```bash
-    wget "https://downloads.d2iq.com/dkp/v2.2.0/dkp-catalog-applications-v2.2.0.tar.gz" -O dkp-catalog-applications.tar.gz
+    wget "https://downloads.d2iq.com/dkp/v2.3.0/dkp-catalog-applications-v2.3.0.tar" -O dkp-catalog-applications.tar
```

Comment:
this will remain `dkp-catalog-applications.tar.gz`, I don't think these git repo archive bundles are intended to change

Reply:
Talked to Grace about catalog apps, but I don't know if this is the same: 
[me] Not sure if the catalog-application image bundles are also not compressed anymore, like in `catalog-applications-image-bundle.tar.gz`
[Grace] those image bundles haven’t been fixed yet but we plan to fix so we can leave those in the docs
Are these image bundles as well or other files?

Reply:
this file here is the `Git repository archive`. 

So yes the catalog applications image bundle will change but this file will stay the same!